src/faebryk/libs/picker/lcsc.py

```diff
@@ -167,15 +164,13 @@ def download_easyeda_info(lcsc_id: str, get_model: bool = True):
     if not footprint_filepath.exists():
         logger.debug(f"Exporting footprint {footprint_filepath}")
         kicad_model_path = (
-            f"{MODEL_PATH}/3dmodels/lcsc.3dshapes"
-            if MODEL_PATH
-            else str(
+            str(
                 "${KIPRJMOD}"
                 / model_base_path_full.relative_to(
-                    config.project.paths.layout.parent, walk_up=True
+                    config.build.paths.layout, walk_up=True
                 )
             )
-            if config.project.paths.layout
+            if config.build.paths.layout
```

Comment:
This line gives the follwing error when using `ato create` `> component > C34572614`

``` python
668 │   def build(self) -> BuildConfig:
669 │   │   if current := self._current_build:
670 │   │   │   return current
671 │   │   raise RuntimeError("No build config is currently active")
```
```bash
RuntimeError: No build config is currently active
```